### PR TITLE
Making images more responsive

### PR DESCRIPTION
**Pull request by Tamir198**
@erluxman this is the single pull request that you have asked for. All the image changes in one pr.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,33 +18,41 @@ It also has shorthand assignment when it's null.
...
+<img src="assets/04cascadeafter.png" height = "600">
 
 ## #Day5 Dart `data class`
 
 Dart does not support data class by default, but with plugins, we can simply generate data class (`copyWith()`,`fromMap()`, `toMap()`, `Named Constructor`, `toString()`,`hashCode()` & `equals()` methods implemented by the tool).
 
-### `🚨❗️Caution❗️🚨` : **Your cursor should be inside the class that you want to generate data class.**
+### `🚨❗️Caution❗️🚨` 
+
+**Your cursor should be inside the class that you want to generate data class.**
```
Comment (erluxman) [inline README.md:56]: This shouldn't be changed .

File: README.md [hunk 2]
```diff
@@ -16,33 +18,41 @@ It also has shorthand assignment when it's null.
 
 `abc??=5 //assigns 5 to abc if it's null`
 
-![null](assets/02ifnull.png)
+
+
+<img src="assets/02ifnull.png" height = "900">
```
Comment (erluxman) [inline README.md:23]: Why a lot of empty lines?

File: week06.md [hunk 3]
```diff
@@ -68,7 +68,10 @@ Then, Flutter will handle the rest.
           )
 [try in codepen](https://codepen.io/erluxman/pen/xxwJRBQ)
 
-![animatedswitcher](assets/37animatedswitcher.gif)
+
```
Comment (erluxman) [inline week06.md:71]: Empty line has inconsistency.. Please Use Markdown Linter. 

If that is not the case, use a line above and below the tag.

File: week07.md [hunk 4]
```diff
@@ -118,6 +120,8 @@ From dart 2.3 onwards, we can use `collection if` and `collection for` operators
 2. ___`A lambda.`___
 
  ![lambda](assets/45lambda.png)
+
```
Comment (erluxman) [inline week07.md:123]: May be it should be changed as well?

File: week10.md [hunk 5]
```diff
@@ -44,6 +44,9 @@ __`To setup lint in Flutter :`__
 2. Add `analysis_options.yaml` in project root directory.
 ![lint](assets/65lint.png)
 
+
```
Comment (erluxman) [inline week10.md:47]: May be here as well?

File: week13.md [hunk 6]
```diff
@@ -189,7 +193,7 @@ PressableDough(
 
 [visit dough](https://pub.dev/packages/dough#-installing-tab-)
 
-![dough](assets/91doughh.gif)
```
Comment (erluxman) [inline week13.md:192]: I like this change if it looks good in real..

**Comment by erluxman** [comment]
So what is your intent? You want to change the dimension of every image or particular file only?

If you want to change for all, you might not to spend some time on 😉

Please also, don't put unnecessary empty lines.

File: README.md (hunk 2 above)
Comment (Tamir198) [reply README.md:23]: You are correct, I can remove those

File: week06.md (hunk 3 above)
Comment (Tamir198) [reply week06.md:71]: I will read about Markdown Linter.

File: week07.md (hunk 4 above)
Comment (Tamir198) [reply week07.md:123]: Are you referring to the image size?

**Comment by Tamir198** [comment]
@erluxman  *You want to change the dimension of every image or particular file only?*.

I wanted to help and give the images with a very large size smaller ( you have to scroll because the image does not fit well do the screen)

*don't put unnecessary empty lines* - good point actually, I was mainly looking at the preview of the images and not how it looks in the text file - point taken.

I will try to fix the problems soon and update my pr.

File: week13.md (hunk 6 above)
Comment (Tamir198) [reply week13.md:192]: I am sorry but I didn't understand what you meant.

[This is how it looks in real](https://github.com/Tamir198/awesomefluttertips/blob/master/week13.md)

File: README.md (hunk 2 above)
Comment (Tamir198) [reply README.md]: @erluxman 
Removed empty lines

File: week07.md (hunk 4 above)
Comment (erluxman) [reply week07.md:123]: Yeah..

File: week13.md (hunk 6 above)
Comment (erluxman) [reply week13.md:192]: Just replacing the md image tag with HTML image tag and not adding any extra empty lines or spaces.

File: README.md [hunk 7]
```diff
@@ -4,7 +4,9 @@
 
 We can type `stless` and `stful` and we get Autocomplete Suggestion to generate Stateless Flutter Widget or Stateful Flutter Widget Respectively.
 
-![statful](assets/01stlesstful.gif)
+
```
Comment (erluxman) [inline README.md:7]: Empty lines here

File: README.md [hunk 8]
```diff
@@ -16,33 +18,39 @@ It also has shorthand assignment when it's null.
...
+<img src="assets/02ifnull.png" height = "900">
 
 ## #Day3 Inner Function
 
 We can define a function inside another function.
 
 This is to encapsulate the inner function from everything else outside the outer function.
 
-![functions](assets/03functions.png)
+
```
Comment (erluxman) [inline README.md:29]: Empty line

File: README.md [hunk 9]
```diff
@@ -16,33 +18,39 @@ It also has shorthand assignment when it's null.
...
 try in [Dartpad](https://dartpad.dartlang.org/290e17306b745ed83b9242653ca55041)
 
-![cascade](assets/04cascadebefore.png)
+
+<img src="assets/04cascadebefore.png" height = "900">
 
 Can be replaced with
 
-![cascadeafter](assets/04cascadeafter.png)
+
```
Comment (erluxman) [inline README.md:43]: empty line

File: README.md [hunk 10]
```diff
@@ -16,33 +18,39 @@ It also has shorthand assignment when it's null.
...
+<img src="assets/03functions.png" height = "500">
 
 ## #Day4 ..Cascade..Chaining..Fluent API
 
 We can chain method/member calls without returning `this` from **method(), getter() and setter()** using cascade operator (..)
 
 try in [Dartpad](https://dartpad.dartlang.org/290e17306b745ed83b9242653ca55041)
 
-![cascade](assets/04cascadebefore.png)
+
```
Comment (erluxman) [inline README.md:38]: empty line

File: README.md [hunk 11]
```diff
@@ -16,33 +18,39 @@ It also has shorthand assignment when it's null.
...
+<img src="assets/04cascadeafter.png" height = "600">
 
 ## #Day5 Dart `data class`
 
 Dart does not support data class by default, but with plugins, we can simply generate data class (`copyWith()`,`fromMap()`, `toMap()`, `Named Constructor`, `toString()`,`hashCode()` & `equals()` methods implemented by the tool).
 
-### `🚨❗️Caution❗️🚨` : **Your cursor should be inside the class that you want to generate data class.**
+### `🚨❗️Caution❗️🚨` 
+
```
Comment (erluxman) [inline README.md:52]: You converted a single line heading into multiple and it won't show the same

File: week03.md [hunk 12]
```diff
@@ -14,7 +14,9 @@ but, Flexible uses `fit :FlexFit.loose` by default.
 
 **flex** = The factor of space taken from parent. Mostly not fully used if `flex: FlexFit.loose` used i.e. `Flexible`.
 
-![flex](assets/15flexibleexpanded.png)
+
```
Comment (erluxman) [inline week03.md:17]: Empty lines.. Just put one line above and 1 below the tag

File: week03.md [hunk 13]
```diff
@@ -51,7 +53,9 @@ You can play with values of snap, floating, pinned etc to get desired effect
 
 ## #Day18 What the Key
 
-![keys](assets/18keys.gif)
+
+
```
Comment (erluxman) [inline week03.md:57]: Here as well

File: week04.md [hunk 14]
```diff
@@ -58,7 +58,7 @@ Ever wanted rectangular/stadium Fab with Notch?
 
 [try this code on your editor](https://gist.github.com/erluxman/fd442639bcaf84e14b31f70b00c48fe9)
 
-![rectangular notch](assets/24rectangularnotch.png)
```
Comment (erluxman) [inline week04.md:61]: This is the way I like it, perfect 👍🏻

File: week05.md [hunk 15]
```diff
@@ -15,8 +15,11 @@ Use Wrap like you use Column or Row just give it's direction (either vertical or
         spacing: 20, //space between previous and next item
         runSpacing: 20, //space between new row or column
         );
+        
 [try in codepen](https://codepen.io/erluxman/pen/YzyENpR)
-![wrap](assets/29wrap.gif)
+
+<img src="assets/29wrap.gif" height ="600">
+
```
Comment (erluxman) [inline week05.md:22]: 2 Empty line below

File: week05.md [hunk 16]
```diff
@@ -197,10 +202,14 @@ VSCode: typing Open Dev Tools in command Pallet.
 
 Learining to use Dart Dev tool  is very 🚨important skill🚨 to have as a Flutter/dart developer.
 
-![Opening in Android Studio](assets/35as.png)
+
```
Comment (erluxman) [inline week05.md:205]: Empty line here

File: week05.md [hunk 17]
```diff
@@ -197,10 +202,14 @@ VSCode: typing Open Dev Tools in command Pallet.
...
+<img src="assets/35as.png" height ="400">
+
 _Opening in Android Stuido_
 
-![vscode](assets/35vscode.png)
+
```
Comment (erluxman) [inline week05.md:210]: Empty lines here

File: week06.md [hunk 18]
```diff
@@ -68,7 +68,7 @@ Then, Flutter will handle the rest.
           )
 [try in codepen](https://codepen.io/erluxman/pen/xxwJRBQ)
 
-![animatedswitcher](assets/37animatedswitcher.gif)
```
Comment (erluxman) [inline week06.md:71]: This is perfect

File: week06.md [hunk 19]
```diff
@@ -103,7 +103,9 @@ All those callbacks behave like `onClick(){}` on Button.
 
 [try on codepen](https://codepen.io/erluxman/pen/wvKxVrE)
 
-![gesture](assets/38gesture.gif)
+
+
```
Comment (erluxman) [inline week06.md:107]: Extra empty line here

File: week06.md [hunk 20]
```diff
@@ -124,7 +126,11 @@ animated_text_kit provides some cool ways to animate text appearences.
 
 [animated_text_kit package](https://pub.dev/packages/animated_text_kit#-installing-tab-)
 
-![animatedtext](assets/39animatetext.gif)
+
+
```
Comment (erluxman) [inline week06.md:130]: Lots of empty lines here

File: week06.md [hunk 21]
```diff
@@ -138,7 +144,9 @@ We can use AnimatedIcon in 5 simple steps.
 
 [play on codepen](https://codepen.io/erluxman/pen/PoPyNrM)
 
-![animatedicon](assets/40animatedicon.gif)
+
```
Comment (erluxman) [inline week06.md:147]: Here as well

File: week06.md [hunk 22]
```diff
@@ -189,8 +197,12 @@ We can use ShapeBorder to give outline to widgets or Clip them on it's shape. Th
 
 [try in codepen](https://codepen.io/erluxman/pen/vYNQLPx)
```
Comment (erluxman) [inline week06.md:199]: 4 empty lines here, 3 extra

File: week07.md (hunk 4 above)
Comment (erluxman) [reply week07.md:123]: I meant extra empty lines

File: week07.md [hunk 23]
```diff
@@ -134,7 +138,9 @@ ColorFiltered works with any widgets as child not just Image.
 
 [get the code](https://gist.github.com/erluxman/7b5c1dfec4461b147d9b00a86d080bb5)
 
-![filters](assets/46colorfiltered.gif)
+
```
Comment (erluxman) [inline week07.md:141]: Empty lines

File: week07.md [hunk 24]
```diff
@@ -161,7 +167,9 @@ Gradients can be easily converted to Shader with createShader() method.
         )
 [get the code](https://gist.github.com/erluxman/b6f1166ac19b7b2654ee2102c58a8837)
 
-![mask](assets/47mask.png)
+
+
```
Comment (erluxman) [inline week07.md:171]: ELS  😀

**Comment by erluxman** [comment]
Please navigate to files changed tab and make sure every change look like this without extra spaces. 

Screen Shot 2020-08-20 at 7 15 30 AM

File: README.md (hunk 11 above)
Comment (Tamir198) [reply README.md]: Fixed

**Comment by Tamir198** [comment]
> Please navigate to files changed tab and make sure every change look like this without extra spaces.
> 
> Screen Shot 2020-08-20 at 7 15 30 AM

I have done it, removing every extra empty line.

Thank you for the tip, this looks better without the extra empty lines 🥇

File: week09.md [hunk 25]
```diff
@@ -122,6 +123,8 @@ __`For VS code`__ : [Pubspec Assist](https://marketplace.visualstudio.com/items?
 :-------------------------:|:-------------------------:
 ![a](assets/62pubspectassist.gif)  |  ![a](assets/62FlutterEnhancement.gif)
 
+
+
```
Comment (erluxman) [inline week09.md:127]: Just this few empty lines

**Comment by erluxman** [comment]
Except the above everything looks dope 😎

File: week09.md (hunk 25 above)
Comment (Tamir198) [reply week09.md]: Fixed 💯

**Comment by erluxman** [comment]
Thanks a lot for your contribution bro 🙏🏼

**Pull request merged**